code/modules/mob/living/carbon/human/human.dm

```diff
@@ -368,33 +368,41 @@
 			var/pocket_id = (pocket_side == "right" ? slot_r_store : slot_l_store)
 			var/obj/item/pocket_item = (pocket_id == slot_r_store ? r_store : l_store)
 			var/obj/item/place_item = usr.get_active_hand() // Item to place in the pocket, if it's empty
-
 			var/delay_denominator = 1
-			if(pocket_item && !(pocket_item.flags&ABSTRACT))
-				if(pocket_item.flags & NODROP)
-					usr << "<span class='warning'>You try to empty [src]'s [pocket_side] pocket, it seems to be stuck!</span>"
-				usr << "<span class='notice'>You try to empty [src]'s [pocket_side] pocket.</span>"
-			else if(place_item && place_item.mob_can_equip(src, pocket_id, 1) && !(place_item.flags&ABSTRACT))
-				usr << "<span class='notice'>You try to place [place_item] into [src]'s [pocket_side] pocket.</span>"
-				delay_denominator = 4
-			else
-				return
-
-			if(do_mob(usr, src, POCKET_STRIP_DELAY/delay_denominator)) //placing an item into the pocket is 4 times faster
-				if(pocket_item)
-					if(pocket_item == (pocket_id == slot_r_store ? r_store : l_store)) //item still in the pocket we search
-						unEquip(pocket_item)
+			if(ishuman(usr))
+				var/mob/living/carbon/human/H = usr
+				if(H.gloves && istype(H.gloves,/obj/item/clothing/gloves/pickpocket))
+					delay_denominator = 3
+				if(pocket_item && !(pocket_item.flags&ABSTRACT))
+					if(pocket_item.flags & NODROP)
+						usr << "<span class='warning'>You try to empty [src]'s [pocket_side] pocket, it seems to be stuck!</span>"
+					usr << "<span class='notice'>You try to empty [src]'s [pocket_side] pocket.</span>"
+				else if(place_item && place_item.mob_can_equip(src, pocket_id, 1) && !(place_item.flags&ABSTRACT))
+					usr << "<span class='notice'>You try to place [place_item] into [src]'s [pocket_side] pocket.</span>"
+					delay_denominator = 4
 				else
-					if(place_item)
-						usr.unEquip(place_item)
-						equip_to_slot_if_possible(place_item, pocket_id, 0, 1)
+					return
 
-				// Update strip window
-				if(usr.machine == src && in_range(src, usr))
-					show_inv(usr)
-			else
-				// Display a warning if the user mocks up
-				src << "<span class='warning'>You feel your [pocket_side] pocket being fumbled with!</span>"
+				if(do_mob(usr, src, POCKET_STRIP_DELAY/delay_denominator)) //placing an item into the pocket is 4 times faster
+					if(pocket_item)
+						if(pocket_item == (pocket_id == slot_r_store ? r_store : l_store)) //item still in the pocket we search
+							unEquip(pocket_item)
+							if(H.gloves && istype(H.gloves,/obj/item/clothing/gloves/pickpocket))
```

Comment:
This'll get fixed once I de-snowflake the code tomorrow the same way I got it to work with non-humans in living.dm. I had assumed that non-humans couldn't strip pockets since the file is human.dm but looking at it now, I'm pretty sure its because humans are the only mobs with pockets.